### ISSUE TITLE
Keka failed to create .tzb-archinve

### ISSUE THREAD

**Issue by justmeMuc** [aonez/Keka#426]
I tried to create a - well - big archive with maximim settings regards compression and resources use with bzip2.

Keka created the archive without any failues or messages regards creating errors, but I hat to find out, that some files are not included in the archive.

While tar tvfz (and xvfz) simply ignores somes files, iZip claimed the file to be corrupted - the archive is not readable.

Thats really anoying as a currupted archive is not worth anything, as files went lost.

Keka Version is 1.1.17 (build 3323) 

Logfiles showing errors could not be found in the system log - thus I cannot send them with this isue.

**Comment by aonez** [comment]
You can't extract it with Keka?

**Comment by justmeMuc** [comment]
I'm sorry, the archive is too big to share it - it's about 148 GB large.
I tried to archive my virtual machines for a "backup" and tried to shrink the archive as great as possible.
So I choosed the following options for BZIP2: 
Method: slowest
Archive as a single file.

CPU-Load was high, well as exceptet ;) - and I got one .tbz-Archive.
I tried to verify with tar -tvfz and this gave some empty folders, exactly - and identically - as Keka did

Any ideas or tips which are a NoGo for that circumstances??

**Comment by aonez** [comment]
Thanks for the feedback @justmeMuc! What kind of virtual machines are those? Parallels? VirtualBox? Also would be awesome to know what files are missing, to see if there's a pattern.

**Comment by justmeMuc** [comment]
Hi @aonez 
Currently it looks like there have some folders beeing completly unreadable by keke - and tar, too.
I tried to back up some of my VirtualBox machines, some are 100% OK, otheres are completly missing and there's only a empty folder available.

As soon as my machine has completed a tar tvfz, I'll post the resultes below.
Kind regards

**Comment by aonez** [comment]
Thanks again @justmeMuc! I'll do some test with VirtualBox files.

**Comment by justmeMuc** [comment]
Hi @anoez, I tried it again - this time it looks much better - the file itself is biggert, too. ;)
The only difference between the two runs is: 1st time a backed up from an APFS to the same disk, the 2nd run was with HFS+ - I don't know if there are limitations regarding the file size or something like that. I'm still useing High Sierra, latest updates for some reasons..
At least, here's the file with the empty folders
[result.txt](https://github.com/aonez/Keka/files/3378393/result.txt)

**Comment by justmeMuc** [comment]
well, here's the result of the 2nd backup (some folders are still empty, as they went lost)
But, as mentioned above, it looks much better with HFS+
[result2.txt](https://github.com/aonez/Keka/files/3379045/result2.txt)

**Comment by aonez** [comment]
> 1st time a backed up from an APFS to the same disk, the 2nd run was with HFS+

That's interesting! I suppose you have enough free space so the compression will fit?

I did some tests in macOS 10.14 Mojave with VirtualBox machines, both while stopped and running but there was no difference, all went ok. My tests where with data about 50GB. I'll do some some more tests in macOS 10.13 High Sierra, using APFS and HFS+.

**Comment by justmeMuc** [comment]
Hi @aonez 
I think this issue can be closed as it seems my filesystem was cranky: 
After deleting some "bigger downloads" the free space was increased only after I rebooted my machine - that's been nuts!

Right now, I'm back to HFS+ with High Sierra - and all runs like a charme. :)

Sorry for any inconvenience.

Kind regards

For your information: I use some non-Apple SSD's in my Mac Pro 5,1 and after some trouble I've disabled TRIM after I had some issues with AFS and found an article in a german Mac-Page with that recomendation.. :(
